packages/cli/src/ceramic-daemon.ts

```diff
@@ -48,6 +58,53 @@ interface MultiQueries {
   queries: Array<MultiQuery>
 }
 
+const ACCESS_LOG_FMT = 'ip=:remote-addr ts=:date[iso] method=:method path=:url http_version=:http-version req_header:req[header] status=:status content_length=:res[content-length] content_type=":res[content-type]" ref=:referrer user_agent=:user-agent elapsed_ms=:total-time[3]';
+
+const makeExpressMiddleware = function (config: LoggerConfig) {
+  const logger = LoggerProvider.makeServiceLogger("http", config)
```

Comment:
`("access", config)` ? To stay consistent with CAS. Also don't want to confuse this with http-client log file.

Reply:
What's the http-client log file?  Also really feels like the fact that these are all http requests getting logged is pretty relevant.  How about "http-access" as the service name?

Reply:
Oh if we were to add logging to the http-client, but I guess that naming wouldn't be relevant.

"http-access" sounds good to me